ladybug_vtk/_extend_sunpath.py

```diff
@@ -70,7 +77,10 @@ def sunpath_to_vtkjs(self, output_folder: str = '.', file_name: str = 'sunpath',
     ticks_minor = compass.ticks_from_angles(angles=compass.MINOR_AZIMUTHS)
     ticks_polydata = [tick.to_polydata() for tick in ticks_major+ticks_minor]
     base_polydata.extend(ticks_polydata)
-    base_dataset = ModelDataSet(name='base_circle', data=base_polydata, color=Color())
+    base_dataset = DisplayPolyData(
+        name='Sun path::Base Circle', identifier='base_circle',
```

Comment:
This it totally a naming/styling thing but we called the "Base Circle" a "Compass" in virtually all of ladybug-core and the Grasshopper plugin. I think the name comes originally from the class you wrote for it in legacy and I think it's a particularly good name for this thing that shows the cardinal directions on all of the climate graphics (not just sun path but also wind rose, radiation dome, etc.)